src/Components/WishList/WishList.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import RecoProductList from '../RecoProductList/RecoProductList';
+import EmptyList from './EmptyList/EmptyList';
+import SelectList from './SelectList/SelectList';
+import ThemeListImg from './ThemeListImg/ThemeListImg';
+import { BASE_URL } from '../../config';
+import './WishList.scss';
+
+const WishList = () => {
+  const [wishProducts, setWishProducts] = useState([]);
+  const [imgSource, setImgSource] = useState(
+    'https://images.unsplash.com/photo-1557682250-33bd709cbe85?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1429&q=80'
+  );
+  const token = localStorage.getItem('token');
+
+  useEffect(() => {
+    fetch(`${BASE_URL}/users/wishlist`, {
+      headers: {
+        Authorization: token,
+      },
+    })
+      .then(res => res.json())
+      .then(data => {
+        setWishProducts(product => (product = data.wish_list));
+      });
+  }, []);
+
+  const handleImageSource = e => {
+    setImgSource(e.target.src);
+  };
+
+  return (
+    <div className="wishList">
+      <div className="header">
+        <img src={imgSource} alt="source" />
+        <div className="titleContainer">
+          <p className="titleTxt">My Wish List</p>
+          <p className="share">Share</p>
+          <div>
+            <button className="listBtn">
+              <i className="icon fas fa-link" />
+            </button>
+            <button className="listBtn">
+              <i className="icon fa fa-envelope" />
+            </button>
+            <button className="listBtn">
+              <i className="icon fab fa-facebook-square" />
+            </button>
+            <button className="listBtn">
+              <i className="icon fab fa-twitter-square" />
+            </button>
+          </div>
+          <p className="infoTxt">
+            Your wish list will be temporarily saved for 7 days.
+          </p>
+          <p className="signInTxt">
+            <Link to="/login" className="linkToLogin">
+              Sign In
+            </Link>
+            &nbsp;to save or share this wish list.
+          </p>
+        </div>
+      </div>
+      <div className="listSelect">
+        <div className="selectLayout">
+          <button className="selectTheme">
+            <span className="themeTxt">
+              Select Your Theme
+              <i className="far fa-arrow-alt-circle-up" />
+            </span>
+          </button>
+          <div className="productNum">1 Product</div>
+        </div>
+        <div className="theme">
+          <ul className="themeList">
+            {WISHLIST_IMG_LIST.map(img => {
```

Comment:
데이터 양이 많지 않은 경우에는 바로 매개변수 자리에서 구조분해 할당하는게 가독성에도 좋습니다:)

```suggestion
            {WISHLIST_IMG_LIST.map(({id, src}) => {
```

Reply:
네 수정하였습니다!